var/spack/repos/builtin/packages/orc/package.py

```diff
@@ -27,7 +28,7 @@ class Orc(CMakePackage):
     depends_on("snappy@1.1.7:")
     depends_on("lz4@1.7.5:")
 
-    patch("thirdparty.patch")
+    patch("thirdparty.patch", when="@1.6")
```

Comment:
Can you change the `thirdparty.patch` filename? Otherwise patch cache issues can happen.

Reply:
> Can you change the thirdparty.patch filename? Otherwise patch cache issues can happen.

@payerle: Can you apply the change that haampie requested?